src/main/java/com/minecolonies/coremod/network/messages/BuildToolPlaceMessage.java

```diff
@@ -133,30 +145,32 @@ public void messageOnServerThread(final BuildToolPlaceMessage message, final Ent
      * @param buildPos The location the hut is being placed.
      */
     private static void handleHut(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 27 to the 20 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)